### DISCUSSION TITLE
Adding users as collab

### DISCUSSION THREAD

**Discussion by ahoneybun** [cosmic-utils/discussions#2] [General]
We have added some more control over repos so that folks have more control over who can access the repos. I think it may be better to make it so that we are all collabs since I don't think we can restrict it to just making PRs/branches?

**Comment by edfloreshz** [comment]
Ideally we should be allowed to make branches and PRs from within all repositories, but that requires custom roles, we don't have that so we need to allow specific users as collabs if we want to have something similar, I think protecting the main branch would accomplish that.

**Reply by ahoneybun** [reply]
Yea I think that is the best that we can do?

**Reply by ahoneybun** [reply]
Alright we have the following team:

https://github.com/orgs/cosmic-utils/teams/cosmic-utils-developers

Which you can add to your project if you want folks to have access to make PR's directly rather than with forks.

**Reply by bhh32** [reply]
That link gives me a 404. I’d like to be able to add my Tailscale applet to this project. https://github.com/bhh32/gui-scale. It’s in the gui-scale-applet part of the workspace.

**Reply by ahoneybun** [reply]
I think you need to be in it perhaps. I sent an invite to the general org to you.
